onnxruntime/core/providers/openvino/backend_utils.cc

```diff
@@ -69,6 +69,43 @@ CreateCNNNetwork(const Provider_ModelProto& model_proto, const GlobalContext& gl
     ORT_THROW(log_tag + "[OpenVINO-EP] Unknown exception while importing model to nGraph Func");
   }
 
+  ValidateCNNNetwork(global_context, subgraph_context, const_outputs_map, ng_function);
+
+  try {
+    return std::make_shared<InferenceEngine::CNNNetwork>(ng_function);
+  } catch (const InferenceEngine::details::InferenceEngineException& e) {
+    ORT_THROW(log_tag + " Exception thrown while making IE::CNNNetwork: " + e.what());
+  } catch (...) {
+    ORT_THROW(log_tag + " Exception thrown while making IE::CNNNetwork");
+  }
+}
+
+InferenceEngine::CNNNetwork ReadCNNNetwork(const Provider_ModelProto& model_proto, const SubGraphContext& subgraph_context, InferenceEngine::CNNNetwork cnn_network) {
+std::string subgraph_name = subgraph_context.subgraph_name;
+#ifndef NDEBUG
+  if (IsDebugEnabled()) {
+    DumpOnnxModelProto(model_proto, subgraph_name + "_static.onnx");
+  }
+#endif
+  InferenceEngine::Core ie;
```

Comment:
Please don't create another instance of InferenceEngine Core. We have to create the InferenceEngine core only once. This is created in the globalcontext. The way we can use it here is global_context_.ie_core

Reply:
Done